lumen/ai/coordinator.py

```diff
@@ -632,227 +600,13 @@ class Planner(Coordinator):
 
     prompts = param.Dict(
         default={
-            "context": {
```

Comment:
Let the planner sort it out because it'll likely invoke TableLookup if it needs context.